### PR TITLE
Adding "Vault App" (Accepted)

### PR DESCRIPTION
**Pull request by geetanshatrey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,4 @@ To add your own project below, just [edit](https://github.com/wwdc/2022/edit/mai
 | Name | Source |    Videos    | Technologies | Status |
 |------|--------|--------------|--------------|--------|
 |Sample Entry|[GitHub](https://github.com/wwdc/2022)|[YouTube](https://youtu.be/dQw4w9WgXcQ)|SwiftUI, AVFoundation|Submitted|
+|[Geetansh Atrey](https://github.com/geetanshatrey)|[GitHub](https://github.com/geetanshatrey/Vault)|-|SwiftUI, CryptoKit|Accepted|
```
Comment (thisIsTheFoxe) [inline README.md:18]:
```suggestion
|[Geetansh Atrey](https://github.com/geetanshatrey)|[GitHub](https://github.com/geetanshatrey/Vault)||SwiftUI, CryptoKit|Accepted|
```

**Review by thisIsTheFoxe — CHANGES_REQUESTED** [review]
Congrats on getting accepted! 🎉
Looks good, so far. But if there's no entry please just leave the field empty. 
Thanks.

**Comment by geetanshatrey** [comment]
> Congrats on getting accepted! 🎉 Looks good, so far. But if there's no entry please just leave the field empty. Thanks.

Thanks a lot! Also, I have committed the changes you had requested.

**Comment by thisIsTheFoxe** [comment]
Great, thank you :)
Please note that as we merge other entries it might cause conflicts which you can resolve right here on GitHub. Just keep in mind the alphabetical order. Sorry for the inconvenience.. 🫠

**Comment by geetanshatrey** [comment]
Made the changes you had requested. Can merge now. Thanks!

**Comment by thisIsTheFoxe** [comment]
Hi again,
It seems there's been some whitespace and other formatting changes now (which leads to more conflicts). Please check it again and only add your entry, keeping the rest of the file as-is.
Thanks :)

**Comment by geetanshatrey** [comment]
Hey, sorry for the inconvenience. Can you review it again and check if it's fine now? Thanks!

**Review by thisIsTheFoxe — APPROVED** [review]
No worries looks good. 👍
Just wired that the GitHub UI shows some additions that were made already. iirc that's on purpose bc of how merging/rebasing is handled. But as long as there's no deletion it should be fine ^^

Again, congrats and thanks for your patience and fixing it.. ^-^

**Pull request merged**